### ISSUE TITLE
Is this package legal?

### ISSUE THREAD

**Issue by AnalyzePlatypus** [branchard/fast-speedtest-api#39]
This package uses Fast.com's private API.
(Fast.com is a public site by Netflix. It runs on their infrastructure)

I have not been able to find any evidence that they allow their API to be used by third parties.

Is using this package legal?

**Comment by samundrak** [comment]
Nope, it's not legal. It even asks to use the token which fast.com uses. 
This tool is helpful though for learning and research purpose
